fe/fe-core/src/main/java/org/apache/doris/nereids/cost/CostModelV1.java

```diff
@@ -329,14 +329,16 @@ public Cost visitPhysicalHashAggregate(
             PhysicalHashAggregate<? extends Plan> aggregate, PlanContext context) {
         Statistics inputStatistics = context.getChildStatistics(0);
         double exprCost = expressionTreeCost(aggregate.getExpressions());
+        double aggCostBaseFactor = 3;
```

Comment:
current cost model is sensitive, do we need to do full test on all sf benchmark and user cases?

Reply:
better to measure the influence of this change on ds 10t.